src/linker/Linker/MessageCategory.cs

```diff
@@ -2,15 +2,19 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System;
+
 namespace Mono.Linker
 {
+	[Flags]
```

Comment:
I would not do this - it will be confusing for all the callsites where we specify message category for loggin - where only one value is every allowed.